nemoguardrails/library/domain_hallucination/semantic.py

```
@@ -0,0 +1,175 @@
+
+    # Look for common typosquatting patterns
+    common_owners = {
+        "github",
+        "pytorch",
+        "tensorflow",
+        "google",
+        "microsoft",
+        "facebook",
+        "aws",
+    }
+    for key, repo_item in github_map.items():
```

Ruff (0.15.15)

[warning] Loop control variable `repo_item` not used within loop body

(B007)